BLSQ/openhexa-app
# Pipeline AddOutput mutation, temporary status page

Add a mutation to add output to PipelineRun, from the pipeline. Used to reference a generated file or table in a run, for the user to easily find them. Also add a temporary status page to see what's the cloud is doing.

## Changes

- Add an add output mutation to the pipelines module
- Add a temporary status html page

## How/what to test

- run a pipeline with the IO subsystem, see in the pipeline run
- go to /pipelines/status to check the current DB. You must be admin to access that page
